content/events/agbt-2023.md

```diff
@@ -0,0 +1,30 @@
+---
+author: "AnVIL"
+conference: "AGBT"
+description: "Advances in Genome Biology and Technology (AGBT) delivers a premier experience where heads of labs, institutions, businesses, financial analysts and other high-level stakeholders come together to advance the field and drive game-changing innovation."
+eventType: "Conference"
+featured: true
+location: "Hollywood, FL, USA"
+sessions: [{sessionStart: "6 Feb 2023"}, {sessionStart: "7 Feb 2023"}, {sessionStart: "8 Feb 2023"}]
+timezone: "America/New_York"
```

Comment:
Hi @jpaten I think the timezone for FL will be different to America/New_York?

Reply:
Actually, given I'm not a USA citizen, can you please just check this bold statement of mine (😁) and confirm they share the same time time zone?

Reply:
They're in the same time zone, I'm not sure if there's any particular weirdness so I'll double check